docs/sql/mssql/migration_1.8.0_1.9.0.sql

```diff
@@ -17,3 +17,49 @@ GO
 -- Add encryption_mode column to pa_application_callback table.
 ALTER TABLE pa_application_callback ADD encryption_mode varchar(255) CONSTRAINT DF_pa_application_callback_encryption_mode DEFAULT 'NO_ENCRYPTION' NOT NULL;
 GO
+
+-- Changeset powerauth-java-server/1.9.x/20240704-callback-event-table.xml::1::Jan Pesek
+-- Create a new table pa_callback_event
+CREATE TABLE pa_application_callback_event (id bigint NOT NULL, application_callback_id varchar(37) NOT NULL, callback_data varchar (max) NOT NULL, status varchar(32) NOT NULL, timestamp_created datetime2(6) CONSTRAINT DF_pa_application_callback_event_timestamp_created DEFAULT GETDATE() NOT NULL, timestamp_last_call datetime2(6), timestamp_next_call datetime2(6), timestamp_delete_after datetime2(6), timestamp_rerun_after datetime2(6), attempts int CONSTRAINT DF_pa_application_callback_event_attempts DEFAULT 0 NOT NULL, idempotency_key varchar(36) NOT NULL, CONSTRAINT PK_PA_APPLICATION_CALLBACK_EVENT PRIMARY KEY (id), CONSTRAINT pa_application_callback_id_fk FOREIGN KEY (application_callback_id) REFERENCES pa_application_callback(id));
+GO
+
+-- Changeset powerauth-java-server/1.9.x/20240704-callback-event-table.xml::2::Jan Pesek
+-- Add max_attempts column to pa_application_callback table.
+ALTER TABLE pa_application_callback ADD max_attempts int;
```

Comment:
Why do we alter a table that was just created? We can just add these columns in the `CREATE TABLE`. The PR was not merged yet to `develop`, so it should be safe even I understand the table went through changes.

Reply:
Ah, I see ok.